### ISSUE TITLE
some questions about installation and "demo_posetrack.m"

### ISSUE THREAD

**Issue by lqy5150** [iqbalu/PoseTrack-CVPR2017#18]
Dear professor,when i run the demo_posetrack.m,and there is a error:Undefined function or variable 'pt_cache_cnn_features'.what should i do?Besides,i  installed the caffe (cpu-only),and didn't use CUDA, is it OK?

**Comment by umariqb** [comment]
Yes, using caffe without CUDA is fine. For undefined function error, you should run "startup.m" in the beginning.

**Comment by lqy5150** [comment]
after i run "startup.m",and run "demo_posetrack.m",there is:
Invalid MEX-file '/media/liu/xuexi/Bishe/deepcut/external/caffe/matlab/+caffe/private/caffe_.mexa64':
/media/liu/xuexi/Bishe/deepcut/external/caffe/matlab/+caffe/private/caffe_.mexa64: undefined symbol:
_ZN2cv8imencodeERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKNS_11_InputArrayERSt6vectorIhSaIhEERKSB_IiSaIiEE.
